deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -0,0 +1,75 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: gcp-filestore-csi-controller
+spec:
+  serviceName: "gcp-filestore-csi-driver"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gcp-filestore-csi-driver
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: gcp-filestore-csi-driver
+    spec:
+      serviceAccount: gcp-filestore-csi-controller-sa
+      containers:
+        - name: csi-external-provisioner
+          image: gcr.io/gke-release/csi-provisioner
+          args:
+            - "--v=5"
+            - "--csi-address=/csi/csi.sock"
+            - "--timeout=250s"
+            - "--feature-gates=Topology=true"
+            - "--extra-create-metadata"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-external-resizer
+          image: gcr.io/gke-release/csi-resizer
+          args:
+            - "--v=5"
+            - "--csi-address=/csi/csi.sock"
+            - "--csiTimeout=120s"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-external-snapshotter
+          image: gcr.io/gke-release/csi-snapshotter
+          args:
+            - "--v=5"
+            - "--csi-address=/csi/csi.sock"
+            - "--timeout=300s"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: gcp-filestore-driver
+          # Modify this image for local development with master branch.
```

Comment:
We don't have image overlays for dev?

Reply:
The only thing we are changing in dev is the filestore driver image. It is modified directly in the overlays/dev manifest. If needed we can add images later. PD CSI does a similar layout.